### PR TITLE
euvatin validator and some other minor changes

### PR DESCRIPTION
**Pull request by MarcoSulla**

### CODE REVIEW

File: src/i18n/en.extra.js [hunk 1]
```diff
@@ -6,9 +6,9 @@ Parsley.addMessages('en', {
   minwords: "This value is too short. It should have %s words or more.",
   maxwords: "This value is too long. It should have %s words or fewer.",
   words:    "This value length is invalid. It should be between %s and %s words long.",
-  gt:       "This value should be greater.",
-  gte:      "This value should be greater or equal.",
-  lt:       "This value should be less.",
-  lte:      "This value should be less or equal.",
-  notequalto: "This value should be different."
+  gt:       "This value should be greater of %s.",
```
Comment (marcandre) [inline src/i18n/en.extra.js]: greater *than*, less *than*, equal *to*

File: src/parsley/validator_registry.js [hunk 2]
```diff
@@ -359,7 +359,16 @@ ValidatorRegistry.prototype = {
           return value === refOrValue;
       },
       priority: 256
-    }
+    },
+    euvatin: {
```
Comment (marcandre) [inline src/parsley/validator_registry.js]: Check how `integer` is implemented. You don't need an actual function.

**Comment by marcandre** [comment]
Added a few comments. Also you have a failing test.

File: src/parsley/validator_registry.js (hunk 2 above)
Comment (MarcoSulla) [reply src/parsley/validator_registry.js]: I tried but it fails integration tests with an obscure error.

**Comment by MarcoSulla** [comment]
Done.

File: test/unit/validator_registry.js [hunk 3]
```diff
@@ -191,6 +191,15 @@ describe('ValidatorRegistry', () => {
     expect($('#element').psly().isValid()).to.be(true);
     $('#equalto').remove();
   });
+  it('should have a euvatin validator', () => {
+    expectValidation('',    'euvatin').to.be(true);
```
Comment (marcandre) [inline test/unit/validator_registry.js]: This test is wrong. I'd simply remove it.

**Comment by MarcoSulla** [comment]
In teory it should not pass only if the field is also required?

**Comment by marcandre** [comment]
Validating `""` is not the concern of the regex. Best not test for it.

**Comment by MarcoSulla** [comment]
I removed it but it continues to have the obscure message.

PS: it's a bit confusing that some validators needs the empty string check for `required` compatibility and others not, anyway.

**Comment by marcandre** [comment]
I apologize. Your implementation was correct and I was confused asking you to check `integer` validation. I simply went back to your original implementation and merged it to master.
Great work, thanks again

**Pull request closed without merging**